src/containers/AdminCampaignEdit.jsx

```diff
@@ -123,6 +123,15 @@ class AdminCampaignEdit extends React.Component {
     })
   }
 
+  handleSubmit = async () => {
+    await this.handleSave()
+    this.setState({
+      expandedSection: this.state.expandedSection >= this.sections().length - 1 ||
+        !this.isNew() ?
+          null : this.state.expandedSection + 1
+    }) // currently throws an unmounted component error in the console
```

Comment:
Possibly related to the unmounted component error, when setting state based on previous state, React docs recommend using the updater method: https://facebook.github.io/react/docs/react-component.html#setstate which I think would look something like this:

`this.setState((previousState) => {
      expandedSection: previousState.expandedSection >= this.sections().length - 1 ||
        !this.isNew() ?
          null : previousState.expandedSection + 1
    })`

Reply:
hmmm. This code works, but it still throws an unmounted component error:
```
this.setState((prevState) => ({
      expandedSection: prevState.expandedSection >= this.sections().length - 1 ||
        !this.isNew() ?
          null : prevState.expandedSection + 1
    }))
```